File: 2490-circular-sentence/2490-circular-sentence.py
class Solution:
    def isCircularSentence(self, snt: str) -> bool:
        arr = snt.split()
        for i in range(len(arr)-1):
            if arr[i][-1] != arr[i+1][0]: return False
        return arr[-1][-1]==arr[0][0]

File: 2490-circular-sentence/2490-circular-sentence.java
class Solution {
    public boolean isCircularSentence(String sentence) {
        String[] sen = sentence.split(" ");
        int n = sen.length;
        for(int i=0; i<n; ++i){
            String cur = sen[i];
            String nx = sen[(i+1)%n];
            if(cur.charAt(cur.length()-1) != nx.charAt(0)) return false;
        }
        return true;
    }
}